public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,142 @@
+  Then ...
+  1. [clone](#clone) or [download](#download) the **QuickStart seed repository**.
+  1. [install the `npm` packages](#install-node "What if you don't have node?")
+  1. run `npm start` to launch the sample application
+
+a#clone
+:marked
+  ### Clone
+
+  Perform the _clone-to-launch_ steps with these terminal commands.
+
+code-example(language="sh" class="code-shell").
+  git clone https://github.com/angular/quickstart.git my-project
+  cd my-project
+  npm install
+  npm start
+
+a#download
+:marked
+  ### Download
+  <a href="https://github.com/angular/quickstart/archive/master.zip" title="Download the QuickStart seed repository">Download the QuickStart seed</a>
+  and unzip it into your project folder. Then perform the remaining steps with these terminal commands.
+
+code-example(language="sh" class="code-shell").
+  cd my-project
+  npm install
+  npm start
+
+.l-main-section
+:marked
+  ## What's different about the QuickStart Seed?
+
+  The QuickStart seed contains the same application as the <live-example><i>live code</i> QuickStart</live-example>.
+  But there are _many more files_ in the project folder on your machine, 
+  most of which you can [learn about later](setup-systemjs-anatomy.html "Setup Anatomy").
+  
+  Focus on the following three TypeScript (`.ts`) files in the **`/app`** folder.
+
+.filetree
+  .file app
+  .children
+    .file app.component.ts
+    .file app.module.ts
+    .file main.ts
+
++makeTabs(`
+    setup/ts/app/app.component.ts,
+    setup/ts/app/app.module.ts,
+    setup/ts/app/main.ts
+  `, '', `
+    app/app.component.ts,
+    app/app.module.ts,
+    app/main.ts
+  `)(format='.')
+:marked
+  The [QuickStart](../quickstart.html "Angular QuickStart") sample only shows the `AppComponent` file.
+  it creates the equivalent of `app.module.ts` and `main.ts` internally _for that example only_
+  so the reader can discover Angular without distraction.
+
+  All other guides and cookbooks expect you to have defined these three files explicitly. 
+  Each file has a distinct purpose and evolves independently as the application grows.
+
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="20%")
+  col(width="80%")
+  tr
+    th File
+    th Purpose
+  tr
+    td <code>app.component.ts</code>
+    td
+      :marked
+        Defines the same `AppComponent` as the same as one in the [QuickStart](../quickstart.html).
+        It is the **root** component of what will become a tree of nested components
+        as the application evolves. 
+  tr
+    td <code>app.module.ts</code>
+    td
+      :marked
+        Defines the `AppModule`, the [root module](appmodule.html) that tells Angular how to assemble the application.
+        Right now it declares only the `AppComponent`. 
+        Soon there will be more components to talk about. 
+  tr
+    td <code>main.ts</code>
+    td
+      :marked
+        Compiles the application with the JiT compiler

```

Comment:
I linked to the glossary which doesn't really add more info. I didn't want to go to the AoT guide because that's overwhelming.